js/containers/portal-dashboard/feedback/question-feedback-panel.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Map } from "immutable";
+import { connect } from "react-redux";
+import { updateQuestionFeedback, updateQuestionFeedbackSettings } from "../../../actions/index";
+import { QuestionLevelFeedbackStudentRows } from "../../../components/portal-dashboard/feedback/question-level-feedback-student-rows";
+import { FeedbackQuestionRows } from "../../../components/portal-dashboard/feedback/feedback-question-rows";
+import { FeedbackLevel, ListViewMode } from "../../../util/misc";
+
+interface IProps {
+  activity: Map<string, any>;
+  activityIndex: number;
+  answers: any;
+  autoScores: any;
+  computedMaxScore: number;
+  currentQuestion: Map<any, any>;
+  currentStudentId: string | null;
+  feedbacks: Map<any, any>;
+  feedbackLevel: FeedbackLevel;
+  feedbacksNeedingReview: Map<any, any>;
+  feedbacksNotAnswered: number;
+  isAnonymous: boolean;
+  listViewMode: ListViewMode;
+  numFeedbacksGivenReview: number;
+  numFeedbacksNeedingReview: number;
+  questionFeedbacks: Map<any, any>;
+  rubric: any;
+  settings: any;
+  students: Map<any, any>;
+  updateQuestionFeedback: (answerId: string, feedback: any) => void;
+  updateQuestionFeedbackSettings: (embeddableKey: string, feedbackFlags: any) => void;
+}
```

Comment:
I wonder if we can find types for some of the `any` types that we have in the props list?

Reply:
I also saw a few props that are not currently used, but might be slotted in for future work.  I would just double check if there are any obsolete props so we can keep the props list length down.